src/test/mon/test_election.cc

```diff
@@ -299,6 +301,31 @@ bool Election::election_stable()
   return true;
 }
 
+bool Election::check_leader_agreement()
+{
+  int leader = electors[0]->logic.get_acked_leader();
+  for (auto i: electors) {
+    if (leader != i.second->logic.get_acked_leader()) {
+      return false;
+    }
+  }
+  if (disallowed_leaders.count(leader)) {
```

Comment:
nit, could just put
```c++
return disallowed_leaders.count(leader) == 0;
```

Reply:
There's log output I want, so don't do that?